centrifuge-app/src/components/Report/FeeTransactions.tsx

```diff
@@ -0,0 +1,82 @@
+import { Pool } from '@centrifuge/centrifuge-js'
+import { Text } from '@centrifuge/fabric'
+import * as React from 'react'
+import { formatDate } from '../../utils/date'
+import { formatBalanceAbbreviated } from '../../utils/formatting'
+import { getCSVDownloadUrl } from '../../utils/getCSVDownloadUrl'
+import { useFeeTransactions, usePoolMetadata } from '../../utils/usePools'
+import { DataTable } from '../DataTable'
+import { Spinner } from '../Spinner'
+import { ReportContext } from './ReportContext'
+import { UserFeedback } from './UserFeedback'
+import type { TableDataRow } from './index'
+import { formatPoolFeeTransactionType } from './utils'
+
+export function FeeTransactions({ pool }: { pool: Pool }) {
+  const { startDate, endDate, setCsvData } = React.useContext(ReportContext)
+  const transactions = useFeeTransactions(pool.id, startDate, endDate)
+  const { data: poolMetadata } = usePoolMetadata(pool)
+
+  const headers = ['Date', 'Fee name', 'Transaction type', `${pool ? `${pool.currency.symbol} amount` : '—'}`]
+
+  const data: TableDataRow[] = React.useMemo(() => {
+    if (!transactions) {
+      return []
+    }
+
+    return transactions
+      ?.filter((tx) => tx.type !== 'PROPOSED' && tx.type !== 'ADDED' && tx.type !== 'REMOVED')
+      .map((tx) => ({
+        name: '',
+        value: [
+          formatDate(tx.timestamp.toString()),
+          poolMetadata?.pool?.poolFees?.find((f) => f.id === tx.poolFee.feeId)?.name || '-',
+          formatPoolFeeTransactionType(tx.type),
+          tx.amount ? formatBalanceAbbreviated(tx.amount, pool.currency.symbol) : '-',
```

Comment:
```suggestion
          tx.amount ? formatBalanceAbbreviated(tx.amount, pool.currency.symbol, 3) : '-',
```
looks like this could use some more decimals (defaults to 1)